packages/libs/lambda-at-edge/src/routing/redirector.ts

```diff
@@ -0,0 +1,109 @@
+import { compileDestination, matchPath } from "./matcher";
+import { RedirectData, RoutesManifest } from "../../types";
+import * as http from "http";
+
+/**
+ * Whether this is the default trailing slash redirect.
+ * @param redirect
+ * @param basePath
+ */
+function isTrailingSlashRedirect(redirect: RedirectData, basePath: string) {
```

Comment:
This can me more simple with regex, and prevent lot of 'if'

Reply:
Thanks! Yeah, it's a bit ugly but basically trailing slash redirects are already handled by different logic, so I wanted to skip these default redirects from routes-manifest. I wanted to avoid additional RegExp here for perf reasons, so this might be the best way for now.

Reply:
Ok, i can test it tomorrow morning

Reply:
Thanks, that will be helpful. I will try to clean up the code a bit more and see if there are other optimizations I can make.